test/infrastructure/docker/config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: gcr.io/k8s-staging-capi-docker/capd-manager:dev
+      - image: gcr.io/k8s-staging-capi-docker/capd-manager-amd64:dev
```

Comment:
why is this changing?

Reply:
Hmm...I didn't change that manually. I believe that this change occurred when I ran `make -C test/infrastructure/docker test-e2e`.
It doesn't need to be in this PR though. I'll take it out.